src/Featurizers/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# ----------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License
+# ----------------------------------------------------------------------
+cmake_minimum_required(VERSION 3.5.0)
+
+macro(Impl)
+    # Defining a function here to introduce a new scope for local variables
+    get_filename_component(_this_path ${CMAKE_CURRENT_LIST_FILE} DIRECTORY)
```

Comment:
I'm just curious as I don't know cmake well enough, but where does the `${CMAKE_CURRENT_LIST_FILE}` come from?

Reply:
[cmake provides it](https://cmake.org/cmake/help/latest/variable/CMAKE_CURRENT_LIST_FILE.html).